### DISCUSSION TITLE
Discord Unban appeal

### DISCUSSION THREAD

**Discussion by Nishat080** [pterodactyl/panel#5105] [Support] [answered]
My account was hacked by some hacker which I fixed after some time and got the account secured. Please unban my account. 
Ban reason:- scam link 

Discord id :- nishat69 
User id :- 819916717778141224

**Comment by rowansc1** [comment, marked as answer]
Hi, you were banned due to your account being hacked. We then resolved that with you. 

You were then shortly banned again because you were trying to get support for unsupported installation scripts and lying to the moderators and people trying to help you about using an installation script. 

For support with the unsupported installation scripts, you need to go to the script developers discord instead of ptero, and please don't lie to us, we are volunteers helping people in our free time, and it just makes it harder for us when people lie to us.

**Reply by rowansc1** [reply]
@Nishat080

**Reply by Nishat080** [reply]
Please Unban me again(last time).

**Reply by rowansc1** [reply]
No, your ban is valid. Sorry :)

**Reply by Nishat080** [reply]
I was working for a client. I have officially installed the panel. I didn't know my client installed wings with script. I would have corrected my mistake if you had given me a warning. Give me one last chance. I won't do that again in the future. I've been on your Discord Server since 2021. This is the first time it's gone wrong. Excuse the first and last mistake.

**Reply by teteuziinhoh-cell** [reply]
Fui suspenso

**Reply by teteuziinhoh-cell** [reply]
Só que eu não quebrei as regras

**Reply by teteuziinhoh-cell** [reply]
Apareceu que eu fiz conteúdo violento mas eu não fiz eu nem lembro

**Reply by teteuziinhoh-cell** [reply]
pode me desbanir

**Reply by teteuziinhoh-cell** [reply]
@rowansc1

**Reply by danny6167** [reply]
> pode me desbanir

How can we unban you if we don't even know who you are

**Reply by teteuziinhoh-cell** [reply]
vou passar o id

**Reply by teteuziinhoh-cell** [reply]
id  do discord circolzxz.xp id do usuário 1486562882714865769

**Reply by danny6167** [reply]
> id do discord circolzxz.xp id do usuário 1486562882714865769

No ban exists for that user ID

**Comment by thexnike** [comment]
Hi My account was hacked by some hacker and now I fixed and improve my account security and no this happend again.

Discord id :- gaming_nike
User id :- 1009308900140515358_

**Reply by danny6167** [reply]
You should be able to join now.

**Comment by dkelelkwkekeallmannwkwkwkwwkkw-del** [comment]
Hey my account got ban perma bc I got hacked and the hacker have send s.x pictures I said it in support too my user is night_kron and I don't have the id

**Reply by dkelelkwkekeallmannwkwkwkwwkkw-del** [reply]
@danny6167  Can you sort it out?

**Reply by dkelelkwkekeallmannwkwkwkwwkkw-del** [reply]
@rowansc1 or maybe can you help me

**Reply by rowansc1** [reply]
Hiya,

Can you please send me your discord user ID?

Thanks

**Reply by dkelelkwkekeallmannwkwkwkwwkkw-del** [reply]
Im sorry i don't have it

**Reply by dkelelkwkekeallmannwkwkwkwwkkw-del** [reply]
@rowansc1 
I got it id is 1334967491104997539

**Reply by dkelelkwkekeallmannwkwkwkwwkkw-del** [reply]
@rowansc1 can you send me a messge when you have Done everything?

**Reply by rowansc1** [reply]
Are you sure that is your ID? I cannot see anyone banned with that name or ID

**Reply by dkelelkwkekeallmannwkwkwkwwkkw-del** [reply]
Yes it is

**Reply by dkelelkwkekeallmannwkwkwkwwkkw-del** [reply]
Name night_kron
User id 1334967491104997539

**Reply by dkelelkwkekeallmannwkwkwkwwkkw-del** [reply]
Do you have discord  then add darkbro8163 thats my 2 account there can i send you a Screenshot of my ban account

**Reply by dkelelkwkekeallmannwkwkwkwwkkw-del** [reply]
Idk bc on mobile i cant send a Screenshot here

**Reply by dkelelkwkekeallmannwkwkwkwwkkw-del** [reply]
@rowansc1 and the problem is i got perm ban bc I got hacked you know and i spend money in discord and now I got perm ban bc this hacker im really sad

**Reply by dkelelkwkekeallmannwkwkwkwwkkw-del** [reply]
@danny6167 @thexnike can you guys find my account? Sorry for ping

**Reply by danny6167** [reply]
There is no ban for that account on our server.

**Reply by dkelelkwkekeallmannwkwkwkwwkkw-del** [reply]
Is it unban on a server or unban for discord

**Reply by danny6167** [reply]
Discord

**Reply by dkelelkwkekeallmannwkwkwkwwkkw-del** [reply]
Can you add me on discord i have a other account there can I send you a picture from my profile that got baned

**Reply by dkelelkwkekeallmannwkwkwkwwkkw-del** [reply]
Darkbro8163

**Reply by rowansc1** [reply]
This is for the pterodactyl discord. Are you sure you have been banned from the pterodactyl discord?
This is not the place to ask for a unban from any other server or discord.

**Reply by dkelelkwkekeallmannwkwkwkwwkkw-del** [reply]
I'm baned from discord the app from store

**Reply by rowansc1** [reply]
lol if your actual discord account is banned from discord. We can't help you. I'm not sure why you'd think pterodactyl could help with that? You should contact discord support

**Reply by dkelelkwkekeallmannwkwkwkwwkkw-del** [reply]
I did but they don't answer i searched on Google discord unban then this came and I thought you can help me im sorry

**Reply by rowansc1** [reply]
no problem
